master/getting-started/openstack/installation/redhat.md

```diff
@@ -5,18 +5,17 @@ title: Red Hat Enterprise Linux 7 Packaged Install Instructions
 For this version of Calico, with OpenStack on RHEL 7 or CentOS 7, we recommend
 using OpenStack Liberty or later.
 
-> **NOTE**
->
-> On RHEL/CentOS 7.3, with Mitaka or earlier, there is a Nova
+> **Note**: On RHEL/CentOS 7.3, with Mitaka or earlier, there is a Nova
 > [bug](https://bugs.launchpad.net/nova/+bug/1649527) that breaks Calico
-> operation.  You can avoid this bug by:
+> operation. You can avoid this bug by:
 >
-> - using Newton or later (recommended)
+> - Using Newton or later (recommended),
```

Comment:
Should have "or," at the end of this line too. (It's important IMO to indicate that these are alternate solutions; as opposed to 3 steps that are _all_ needed.)

Reply:
I'm going to defer to you on this one. Grammatically it's a little weird to have two "ors" in a single sentence, however, it's probably more important to ensure that people don't get confused here. 👍